### DISCUSSION TITLE
Show favicons in the library

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#2360] [Ideas]
I think it'd be nice if the library had a better treatment for favicons, so it was clearer which domains the replays are coming from. It would also help put a bit more color into the UI

I hear this API is great for this https://faviconkit.com/

#2161

**Comment by jonbell-lot23** [comment]
March 15 2022 update:

As people's libraries get more busy, features like this will continue to rise in importance. We're scaling!
